rrmngmnt/storage.py

```diff
@@ -0,0 +1,183 @@
+import os
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a %s %s/%s'
+PVSCAN_CACHE_CMD = 'pvscan --cache'
+PVSCAN_CMD = 'pvscan'
+CREATE_FILE_CMD = 'touch %s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to local directory
+
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
+        temporary folder is created and it is the responsible of the
+        caller to remove it at the end
+        :type target
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'vers=3']
+        :type opts: list
+        :return: Path to mount point if succeeded, None otherwise
+        """
+        target = '/tmp/mnt_point' if target is None else target
+        cmd = ['mkdir', target]
+        self.logger.info(
+            "Creating directory %s to use as a mount point", target
+        )
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
```

Comment:
yes, check run_command function, you have there logger.info (by you) and logger error if rc